### PR TITLE
Refactor CSS, Part 1

### LINKED ISSUE #88: Implement new CSS styles
The new designs in Figma show that we'll need to update our CSS. Design has worked on making styles across Cal-ITP websites more consistent, so we have changes to fonts, font-sizes, colors, and spacing.

This ticket does not involve re-ordering or removing any content. It is solely focused on the styles, so you'll need to do a bit of mapping between the old and new designs.

Also note this ticket does not include the navbar or footer -- those are to be done in #93.

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #88 

## What this PR does, by priority
1. New styles: Link styles for all links, including Header and Nav. Adds a few new variables.
2. New image: Adds the mobile-width image for the last railway track image
3. New spacing: Fixes a lot of small spacing issues on Homepage
4. Refactor: Delete unused CSS classes
5. Refactor: Refactors entire #deck CSS section into classes
6. Refactor: Font sizes to rem
7. Refactor: Rename CSS classes to generic
8. Refactor: Move around CSS and comment in sections

CSS file is now under 400 lines.

What is not included?
- Nav Pills, will be done in #92 #91

### CODE REVIEW

File: src/_includes/header.html [hunk 1]
```diff
@@ -21,19 +21,16 @@
       <div class="navbar-collapse collapse justify-content-end" id="mainNav">
         <ul class="navbar-nav mb-0">
           <li class="nav-item">
-            <a
-              class="h4 mb-0 nav-link active"
-              aria-current="page"
-              href="#">About the project</a>
+            <a class="h4 mb-0 nav-link d-block" href="#">About the project</a>
```
Comment (thekaveman) [inline src/_includes/header.html:24]: Will be a conflict with #159

File: src/_includes/header.html [hunk 2]
```diff
@@ -21,19 +21,16 @@
...
-            <a class="h4 mb-0 nav-link" href="{% link resources.html %}">Resources</a>
+            <a class="h4 mb-0 nav-link d-block" href="{% link resources.html %}">Resources</a>
           </li>
           <li class="nav-item">
-            <a class="h4 mb-0 nav-link" href="{% link press.html %}">Press</a>
+            <a class="h4 mb-0 nav-link d-block" href="{% link press.html %}">Press</a>
           </li>
           <li class="nav-item">
-            <a class="h4 mb-0 nav-link" href="#">Reach out</a>
+            <a class="h4 mb-0 nav-link d-block" href="#">Reach out</a>
```
Comment (thekaveman) [inline src/_includes/header.html:33]: Conflict with #159

File: src/index.html [hunk 3]
```diff
@@ -3,39 +3,46 @@
...
-  <picture>
-    <img
-      id="triforce"
-      src="images/hero-header.png"
-      alt="A trio of images, clockwise from top: a bus, a train platform with a sign that announces “Next train in 3 minutes,” and a transit rider paying their fare by tapping their smartphone’s mobile wallet on a payment reader when boarding" />
-  </picture>
+  <div class="col-12 offset-lg-1 col-lg-5 d-flex align-items-center justify-content-center">
+    <picture>
+      <img
+        style="max-width: 441px;"
```
Comment (thekaveman) [inline src/index.html:20]: This feels a little strange, I think the only place we have an inline `style` attribute? Qualms about moving it back into the CSS?

**Review by thekaveman — COMMENTED** [review]
Overall this all looks great. Just one question about the inline style

File: src/index.html (hunk 3 above)
Comment (machikoyasuda) [reply src/index.html:20]: No qualms really!

**Review by thekaveman — APPROVED** [review]
👍

**Pull request merged**